cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/analyzer/goals/GoalUtilsTest.java

```diff
@@ -17,75 +17,35 @@ public void testComputeResourceUtilizationBalanceThreshold() {
+    verifyComputingResourceUtilizationBalanceThreshold(resource, true, 0.2, 0.3 * (1 - ((1.3 * 1.2) - 1) * ResourceDistributionGoal.BALANCE_MARGIN));
 
     // Verify 4: Not low utilization and compute balance threshold upper bound
-    verifyComputingResourceUtilizationBalanceThreshold(0.3,
-                                                       resource,
-                                                       ResourceDistributionGoal.BALANCE_MARGIN,
-                                                       false,
-                                                       true,
-                                                       0.2,
-                                                       1.3,
-                                                       1.2,
-                                                       0.3 * (1 + ((1.3 * 1.2) - 1) * ResourceDistributionGoal.BALANCE_MARGIN));
-
+    verifyComputingResourceUtilizationBalanceThreshold(resource, false, 0.2, 0.3 * (1 + ((1.3 * 1.2) - 1) * ResourceDistributionGoal.BALANCE_MARGIN));
   }
 
-  private void verifyComputingResourceUtilizationBalanceThreshold(double avgUtilizationPercentage,
-                                                                  Resource resource,
-                                                                  double balanceMargin,
+  private void verifyComputingResourceUtilizationBalanceThreshold(Resource resource,
                                                                   boolean isLowerThreshold,
-                                                                  boolean isTriggeredByGoalViolation,
                                                                   double lowUtilizationThreshold,
-                                                                  double resourceBalancePercentage,
-                                                                  double goalViolationDistributionThresholdMultiplier,
                                                                   double expectedComputedBalanceThreshold) {
 
     BalancingConstraint mockBalanceConstraint = EasyMock.mock(BalancingConstraint.class);
     EasyMock.expect(mockBalanceConstraint.lowUtilizationThreshold(resource)).andReturn(lowUtilizationThreshold).anyTimes();
-    EasyMock.expect(mockBalanceConstraint.resourceBalancePercentage(resource)).andReturn(resourceBalancePercentage).anyTimes();
+    EasyMock.expect(mockBalanceConstraint.resourceBalancePercentage(resource)).andReturn(1.3).anyTimes();
     EasyMock.expect(mockBalanceConstraint.goalViolationDistributionThresholdMultiplier()).
-        andReturn(goalViolationDistributionThresholdMultiplier).anyTimes();
+        andReturn(1.2).anyTimes();
 
     EasyMock.replay(mockBalanceConstraint);
-    double computedBalanceThreshold = GoalUtils.computeResourceUtilizationBalanceThreshold(avgUtilizationPercentage,
+    double computedBalanceThreshold = GoalUtils.computeResourceUtilizationBalanceThreshold(0.3,
```

Comment:
Nit: Can we move the hardcoded constants to static variables?

Reply:
Sure